src/bokeh/core/property/pd.py

```diff
@@ -35,6 +39,7 @@
 #-----------------------------------------------------------------------------
 
 __all__ = (
+    'EagerDataFrame',
```

Comment:
while we are at it why don't we rename this file to `data_frame.py` since it is no longer pandas-specific (users are expected to import properties from the top level `bokeh.properties` so I consider changing the module name to be an internal change)